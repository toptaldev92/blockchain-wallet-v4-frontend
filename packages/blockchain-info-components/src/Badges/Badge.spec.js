describe('Badge Component', () => {
  it('should have a passing test', () => {
    expect(true).toEqual(true)
  })
})
