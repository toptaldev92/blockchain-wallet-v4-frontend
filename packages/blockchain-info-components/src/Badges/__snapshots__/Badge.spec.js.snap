// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge component renders correctly 1`] = `<div />`;
